### ISSUE TITLE
[Feature]: Add 'first' to define_metric summary options

### ISSUE THREAD

**Issue by alexhh-inceptive** [wandb/wandb#9936]
### Description

Sometimes (e.g. in finetuning) it's useful to compare performance at step 0 (pretrained model) with later steps. A First summary metric would support this

### Suggested Solution

Add first to the list of supported summaries in define_metric

**Comment by fmamberti-wandb** [comment]
Hi @alexhh-inceptive , thank you for reaching out with your feature request.

I've now raised this with our product team. 

In the meantime, you could log the metrics at step 0 as separate summary metrics, for example with `wandb.summary['<metric_name>_at_step_0'] = <your_metric_value>` so that you can store the metrics as step 0 and compare them with the metric after finetuning.

**Comment by alexhh-inceptive** [comment]
sounds good, thanks!

**Comment by fmamberti-wandb** [comment]
Hi @alexhh-inceptive, we got a [PR](https://github.com/wandb/wandb/pull/10121) merged which adds a 'first' for the `define_metric` summary options. Thank you for raising this and feel free to reach out for any furhter questions.
